config/webpack.prod.js

```diff
@@ -9,6 +9,52 @@ const sortBy = require('lodash/sortBy');
 
 const config = require('./webpack.dev.js')
 
+const uniqueWebpackModules = (data) => {
+  // recursively flatten nested modules
+  const transformModule  = m  => m.modules ? transformModules(m.modules) : m.name.split("!").pop();
+  const transformModules = ms => ms.flatMap(m => transformModule(m));
+
+  var modules = transformModules(data.modules);
+  modules = [...new Set(modules)]; // uniq
+
+  return modules;
+}
+
+const packagesFromModules = (modules) => {
+  // find the package.json file for each module
+  modules = modules.filter(m => m.includes("node_modules"));
```

Comment:
modules is a global here, missing `const`?

Reply:
ugh I hate javascript 😆

Reply:
@himdel Oh wait, is it?  I thought it's just updating the incoming parameter.  I'm not introducing a new variable.

Reply:
oh yeah, you're right `modules` is a param here

Reply:
travis went green so all is good now :)